### PR TITLE
added pisces-db recipe

### PR DESCRIPTION
**Pull request by samuelchodur**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pisces-db/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  name: {{ name|lower }}-db
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pisces-{{ version }}.tar.gz
+  sha256: 9496b9f613348952db07121675fe32f45e23740c9d4e3885738b24ce3778ad66
+
+build:
+  skip: True    # [win]
+  number: 0
```
Comment (xhochy) [inline recipes/pisces-db/meta.yaml:15]: This can be noarch
```suggestion
  number: 0
  noarch: python
```

Comment (samuelchodur) [reply recipes/pisces-db/meta.yaml:15]: Hi, @xhochy. The upstream package specifies the platforms to be, 'Mac OS X', 'Linux/Unix' on line 39 of their [setup.py.](https://github.com/LANL-Seismoacoustics/pisces/blob/feca7afd69da84289819b09b22eb5b152818bc52/setup.py) I was unable to get the win_64 check to pass when using noarch.

Comment (xhochy) [reply recipes/pisces-db/meta.yaml:15]: Failing windows CI with noarch packages is ok. They are only built on Linux on the feedstock. noarch is much less maintainence than non-noarch.

Comment (samuelchodur) [reply recipes/pisces-db/meta.yaml:14]: @xhochy I made some changes based on what you have told me. Some checks were not successful as expected.

**Comment by xhochy** [comment]
The Windows error is actually valid. With noarch packages, you need to declare the Python entry_points explicitly in the meta.yaml (see the conda build documentation for this).

**Pull request merged**